pnpm-lock.yaml

```diff
@@ -36,7 +36,7 @@ importers:
       '@types/node': 14.18.33
       '@typescript-eslint/eslint-plugin': 5.21.0_5yvnzvdwzksrnum37j3gumwy4y
       '@typescript-eslint/parser': 5.21.0_eslint@8.14.0
-      '@vercel/build-utils': link:packages/build-utils
+      '@vercel/build-utils': 6.0.0
```

Comment:
Why is the lockfile changing? Did use the correct version of pnpm?

Reply:
Oh, definitely not expected, will revert